demo/src/app/app.config.ts

```diff
@@ -116,10 +115,6 @@ export class AppConfig extends AbstractAppConfig {
     return this.config.print_service_url;
   }
 
-  public getRemotePrintServiceUrl() {
```

Comment:
I think this makes this a breaking change and so it would need to go to the next major release...

Reply:
I've now re-introduced this in `app.config.ts` as a deprecated, no-op function that returns `undefined`, ahead of its removal in the next major release.